lib-injection/java/docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox
```

Comment:
this file should not be here. This is part of the public API of the Java library and should be maintained there.